pandas/_libs/window.pyx

```diff
@@ -96,280 +96,20 @@ def _check_minp(win, minp, N, floor=None) -> int:
 # Physical description: 366 p.
 #               Series: Prentice-Hall Series in Automatic Computation
 
-# ----------------------------------------------------------------------
```

Comment:
i don't think using _check_minp above? and likely can be moved to indexer.pyx anyhow (next pass)

Reply:
also the references above are misplaced, not sure where they should go